coala_quickstart/generation/Bears.py

```diff
@@ -24,9 +24,29 @@ def filter_relevant_bears(used_languages):
     log_printer = LogPrinter(NullPrinter())
     used_languages.append(("All", 100))
 
-    bears_by_lang = {lang: set(inverse_dicts(*get_filtered_bears(
+    all_bears_by_lang = {lang: set(inverse_dicts(*get_filtered_bears(
         [lang], log_printer)).keys()) for lang, _ in used_languages}
 
+    bear_list = {"C": ["ClangBear",
+                       "ClangComplexityBear",
+                       "CSecurityBear",
+                       "GNUIndentBear"],
+                 "JavaScript": ["JSHintBear",
+                                "JSComplexityBear"],
+                 "Java": ["JavaPMD",
+                          "CheckstyleBear"],
+                 "Python": ["PEP8Bear",
+                            "PyLintBear",
+                            "YapfBear"]}
```

Comment:
make them sets instead of lists and move this into Constants with an upper case variable name?